packages/puppeteer-core/src/bidi/Browser.ts

```
@@ -289,6 +291,29 @@ export class BidiBrowser extends Browser {
     await this.#browserCore.uninstallExtension(id);
   }
 
+  override screens(): Promise<ScreenInfo[]> {
+    throw 'Not implemented';
```

let's `throw UnsupportedOperation()` and in other throw statements as well

---

Done.